config/default/sitenow_dispatch.settings.yml

```diff
@@ -0,0 +1 @@
+api_key: null
```

Comment:
I think we'll want to remove this file as it causes problems with config_ignore after #4091.

Reply:
Removed!